prime-router/src/test/kotlin/fhirengine/azure/FhirFunctionTests.kt

```diff
@@ -726,59 +743,93 @@ class FhirFunctionTests {
             }
         }
 
+        /*
+        Send a FHIR message to an HL7v2 receiver and ensure the message receiver receives is translated to HL7v2
```

Comment:
These are now end-to-end tests (integration) of the translate function that verify the blob is correct (that it either matches the original blob or is translated)